examples/feature_convert/partcad.yaml

```
@@ -26,3 +36,15 @@ parts:
   cube_stl:
     type: stl
     path: stl/cube.stl
+
+  cube_enrich:
+    type: enrich
+    source: :cube_cadquery
+    with:
+      height: 2
+      width: 5
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)